db/migrate/20191003083950_create_payroll_runs.rb

```diff
@@ -0,0 +1,8 @@
+class CreatePayrollRuns < ActiveRecord::Migration[5.2]
+  def change
+    create_table :payroll_runs, id: :uuid do |t|
+      t.string :created_by, null: false
+      t.timestamps
```

Comment:
One thing that has just occurred to me is that we're not adding indexes on our `created_at` timestamps, when we probably should. Can we add one as part of this migration?